### DISCUSSION TITLE
Task description to weekly hours

### DISCUSSION THREAD

**Discussion by Joppefin** [kimai/kimai#4777] [General]
Is there any way to add for example pop up (or some other way) asking task description, when user is adding hours through Weekly hours -page?

Description is basically mandatory field for us, but "Weekly hours" is so much quicker way to add hours than "My times".

**Comment by duyanhtruong** [comment]
@kevinpapst This feature would be very helpful for us as well. Could we consider adding this in the future? Maybe instead of the time options in the dropdown button, we'd be able to add comments/description there?

Something like this:
![image](https://github.com/user-attachments/assets/5c9c6859-f0cd-4921-a0c3-fec82e5113ab)

**Comment by kernie** [comment]
I like the idea. 

But you can bulk edit the timesheets and add the description all at once as a "workaround".
